pinata/pinata.go

```diff
@@ -279,3 +279,12 @@ func (p *PinataCloud) UploadMultiFile(multiFace interface{}) ([]*types.UploadedF
 
 	return uploadedFiles, nil
 }
+
+/*
+DeleteFile deletes a file from a bucket on Google Cloud Storage and returns an error if one occurs.
+
+Note: DeleteFile requires that a default bucket be set in bifrost.BridgeConfig.
+*/
```

Comment:
this comment is wrong....this is Pinata `DeleteFile deletes a file Pinata and returns an error if one occurs.` it does not use buckets.